src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value) {
+      calculator.result += value;
+    },
+
+    subtract(value) {
+      calculator.result -= value;
```

Comment:
Try to rewrite all methods with this. Because if you change object name in line 40, you should change it in all methods. Using this - no such a problems.

```suggestion
      this.result += value;
    },

    subtract(value) {
      this.result -= value;
```